dotCMS/src/main/java/com/dotcms/rest/api/v1/storage/MetadataForm.java

```diff
@@ -0,0 +1,52 @@
+package com.dotcms.rest.api.v1.storage;
+
+import com.dotcms.rest.api.Validated;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import javax.validation.constraints.NotNull;
+
+@JsonDeserialize(builder = MetadataForm.Builder.class)
+public class MetadataForm extends Validated {
+
+    @NotNull
+    private final String  field;
+    private final boolean cache;
+
+    public MetadataForm(final Builder builder) {
+
+        this.field      = builder.field;
+        this.cache      = builder.cache;
+    }
+
+
+    public String getField() {
+        return field;
+    }
+
+    public boolean isCache() {
+        return cache;
+    }
+
+
+
+    public static final class Builder {
+
+        @JsonProperty private String  field;
```

Comment:
Codacy found an issue: [Field field has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6220272)